src/app/components/detailed-page/components/detailed-page-hero-rating/detailed-page-hero-rating.component.ts

```diff
@@ -1,11 +1,23 @@
 import { Component, Input } from '@angular/core';
+import { RatingService } from 'src/app/services/rating.service';
+import { RatingPostConfig } from 'src/app/types/rating-post-config';
 
 @Component({
   selector: 'mdb-detailed-page-hero-rating',
   templateUrl: './detailed-page-hero-rating.component.html',
   styleUrls: ['./detailed-page-hero-rating.component.scss'],
 })
 export class DetailedPageHeroRatingComponent {
+  constructor(private ratingService: RatingService) {}
   @Input() public value = 0;
   @Input() public title = 'Rating: ';
+  @Input() public ratingPostConfig?: RatingPostConfig;
+
+  public showDropdown = false;
+
+  public ratedMaps = this.ratingService.ratedMaps;
```

Comment:
Why do you need whole ratedMaps? Maybe just ask for certain rating

Reply:
Updated, added getRating method to RatingService.